client/src/partials/patients/registry/modal.js

```diff
@@ -0,0 +1,100 @@
+angular.module('bhima.controllers')
+.controller('PatientRegistryModalController', PatientRegistryModalController);
+
+PatientRegistryModalController.$inject = [
+  '$uibModalInstance', 'InventoryService', 'PatientService', 'util'
+];
+
+function PatientRegistryModalController( $uibModalInstance, Inventory, patients, util) {
+  var vm = this;
+
+  vm.period = {
```

Comment:
This looks like a very useful utility!  In the future, we should put this in a service so that many controllers can benefit from it.